kms/api/Program.cs

```diff
@@ -427,6 +504,247 @@ public static object RestoreCryptoKeyVersion(string projectId, string location,
         }
         // [END kms_restore_cryptokey_version]
 
+        // [START kms_get_cryptokey_policy]
+        public static object GetCryptoKeyIamPolicy(string projectId, string location,
+            string keyRing, string cryptoKey)
+        {
+            var cloudKms = CreateAuthorizedClient();
+            // Generate the full path of the parent to use for getting the crypto key IAM policy.
+            var parent = $"projects/{projectId}/locations/{location}/keyRings/{keyRing}/cryptoKeys/{cryptoKey}";
+            try
+            {
+                var result = cloudKms.Projects.Locations.KeyRings.CryptoKeys.GetIamPolicy(parent).Execute();
+                if (result.Bindings != null)
+                {
+                    result.Bindings.ToList().ForEach(response =>
+                    {
+                        Console.WriteLine($"Role: {response.Role}");
+                        response.Members.ToList().ForEach(member =>
+                        {
+                            Console.WriteLine($"  Member: {member.ToString()}");
+                        });
+                    });
+                }
+                else
+                {
+                    Console.WriteLine($"Empty IAM policy found for CryptoKey: {parent}");
+                }
+            }
+            catch (Google.GoogleApiException e)

```

Comment:
The try .. catch code falls inside the doc tags and will therefore appear in the documentation page, which clouds the sample a bit, and it doesn't need to.  We could put the try ... catch in a function higher in the stack.  Or, just remove it entirely (my preference.)  In that case, when it fails, the user still sees a stack trace and message and the return value will be nonzero.  Test code that expects a failure needs to change a little bit.  But test code that expects success would remain unchanged.